src/Core/SessionManager.cpp

```diff
@@ -87,7 +87,8 @@ void SessionManager::restoreSession(const QString &path)
     auto oldSize = app->size();
     app->setUpdatesEnabled(false);
 
-    for (auto const &tab : tabs)
+    for (auto const &tab : tabs) // NOLINT: loop variable 'tab' is always a copy because the range of type 'QJsonArray'
```

Comment:
To suppress a warning put in that line as comment `NOLINT: <you can put here reason as to why you supressed this report instead of fixing it`.